lib/order.rb

```diff
@@ -0,0 +1,68 @@
+require_relative 'menu'
+require_relative 'confirmation'
+require 'dotenv'
+Dotenv.load
+
+class Order
+
+  attr_reader :selection, :menu, :prices, :total, :orders, :confirmation
+
+  def initialize(menu = Menu.new)
+  @menu = menu
```

Comment:
Try and format your code in accordance with a Ruby style guide.  It really helps readability, and also makes it look fully professional.